src/app/pages/server-listing/server-listing.component.scss

```
@@ -1,3 +1,24 @@
+.server-listing-hero-header {
+  @apply flex gap-4 justify-between items-center;
+}
+
+.server-listing-hero-actions {
+  @apply flex flex-wrap items-center gap-3 lg:justify-end;
+}
+
+.server-listing-header-action {
+  @apply inline-flex items-center gap-2 px-3 py-2;
+  min-height: 50px;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)